vitest.shared.config.mts

```diff
@@ -17,7 +17,7 @@ export default defineConfig({
     coverage: {
       enabled: true,
       provider: "istanbul",
-      reporter: ["json", "text", "html"],
+      reporter: ["json", "text"],
```

Comment:
The `html` reporter is helpful locally for spotting uncovered code.

Reply:
I will reimplement it, I had wrongly assumed it wasn't being used.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2512/commits/5656542ba1818383225ff3a37bebbae2fefbe16f